civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php

```diff
@@ -21,11 +21,6 @@
   <div class="report-content panel-pane pane-block chr_panel chr_panel--no-padding">
     <?php if (!empty($jsonUrl)): ?>
       <div class="report-block report-builder tab-pane">
-        <div class="chr_search-result__header">
-          <div class="chr_search-result__total">
```

Comment:
Since you removed the markup, what about the css classes here? Are these classes specific to the header or used in other parts as well?

Reply:
I checked, there are none.

Reply:
@bmnepali these classes are used elsewhere in SSP so the class definitions can't be removed. Examples:

https://github.com/compucorp/civihr-employee-portal/search?utf8=%E2%9C%93&q=chr_search-result__header&type=